src/test/java/com/karankumar/bookproject/backend/service/BookServiceTest.java

```diff
@@ -54,23 +52,20 @@
     @Autowired private BookService bookService;
     @Autowired private CustomShelfService customShelfService;
     @Autowired private TagService tagService;
+    @Autowired private PredefinedShelfService predefinedShelfService;
```

Comment:
Could you please use constructor injection here? I appreciate there was already field injection, but we have already started refactoring our tests to move to constructor injection